src/components/data-management/ProjectDetails.jsx

```diff
@@ -64,15 +71,19 @@ const ProjectDetails = ({ width, height }) => {
   );
   const projects = useSelector((state) => state.projects);
   const experiments = useSelector((state) => state.experiments);
+  const environment = useSelector((state) => state.networkResources?.environment);
   const samples = useSelector((state) => state.samples);
   const { activeProjectUuid } = useSelector((state) => state.projects.meta) || false;
   const activeProject = useSelector((state) => state.projects[activeProjectUuid]) || false;
+  const { backendStatus } = useSelector((state) => state.experimentSettings);
+
   const [tableData, setTableData] = useState([]);
   const [tableColumns, setTableColumns] = useState([]);
   const [sortedSpeciesData, setSortedSpeciesData] = useState([]);
   const [sampleNames, setSampleNames] = useState(new Set());
   const [canLaunchAnalysis, setCanLaunchAnalysis] = useState(false);
   const [analysisModalVisible, setAnalysisModalVisible] = useState(false);
+  const [pipelineHasRun, setPipelineHasRun] = useState(true);
```

Comment:
why is the default `true`? shouldn't we assume that the pipeline has not run?

Reply:
oh yea, this is supposed to be `false`. Changed accordingly,